### ISSUE TITLE
Slow temperature measurment

### ISSUE THREAD

**Issue by daestx** [polihedron/DIY-Reflow-Oven#15]
Yesterday, I put the reflow oven into operation. In principle, the hardware works. However, my temperature measurement is causing me problems. It looks as if the temperature is strongly time-delayed. The latency then of course has an effect on the control behavior.

I suspect that this is due to my sensor. I bought the following kit with a K-sensor.

https://de.aliexpress.com/item/32985892905.html?spm=a2g0o.order_list.order_list_main.19.1e7d5c5fAVa7g2&gatewayAdapt=glo2deu

The sensor has a thread on its probe. I did not remove this as I was worried about damaging the sensor. However, I suspect that the latency is due to the large temperature transient, as additional material has to be heated up.

I did not see this attachment to the sensor tip in the pictures of your oven. Have you removed it? Or did you use a different K temperature sensor?

At the moment my temperature curve looks something like this:

![2024-08-11_19-03_1](https://github.com/user-attachments/assets/4a616c67-7754-4092-85e0-3dcfe927aada)
![2024-08-11_19-03](https://github.com/user-attachments/assets/68a716fc-09a4-44fc-af2c-e213746f7525)

My oven definitely reaches 240 °C. I don't actually think it's the PID controller. The control behavior seems to be OK.

If you'd recommend a different sensor, please add a link.

But maybe you have another idea, why I'm facing this issue.

**Comment by polihedron** [comment]
Hello, I brought the same sensor, 
I've extracted it from the original head for better, more accurate readings, that was quite easy with Proxxon micromot 60e.
try also lower HEAT UP parameter, 0,7C/s or 0,5C/s if you don't have build in fan.

**Comment by daestx** [comment]
I'll try this. Hopefully the temperature sensor is more responsive then. Can you describe in more details how you did this with the Proxxon tool? First I'll try to widen/open the crimping....

And I have exactly the same oven with the build in fan.

**Comment by daestx** [comment]
I have now removed the sensor head. To do this, I simply used a side cutter and pointed pliers and cut the head open up to the crimp. The sensor can then be pulled out of the sensor head by gently wiggling the cable.

And what can I say. I wouldn't have thought that 3.34g of steel would have such an effect. In any case, the oven now works as it should. Here are a few more pictures:

![2024-08-13_21-39_1](https://github.com/user-attachments/assets/752568e4-8763-42a1-bcbd-3028d8617cb2)
![2024-08-13_21-39](https://github.com/user-attachments/assets/75e5cfdb-b66e-4335-acee-c0d15c0eefee)
![2024-08-13_21-38](https://github.com/user-attachments/assets/c158fe2f-bdae-4162-a17c-027014f796be)


I guess this topic can be closed.

I'm really happy. Thank you for this cool project. Now I can start to solder some PCBs ;-)
